src/test/java/com/github/shredder121/gh_event_api/handler/AbstractHandlerTest.java

```diff
@@ -76,6 +77,21 @@
         }
     }
 
+    private static GitHub getGitHub() throws IOException {
+        GitHubBuilder builder;
+        try {
+            builder = GitHubBuilder.fromCredentials();
+        } catch (IOException noAuthorizationDetails) {
+            // This is fine, and still gives you 60 requests per hour, so one-off test runs still work
+            builder = new GitHubBuilder();
+        }
+        OkHttpClient client = new OkHttpClient();
+        client.setCache(new Cache(Paths.get(".", ".cache").toFile(), FileUtils.ONE_MB * 10));
```

Comment:
Indeed!
Thanks for thinking along though.

Do you think that 10MB is too much? It's only for ~20 files, and they are not that big.
But github.developer.com repository, /user, I think could also be cached.